machine_management/cluster_api_machine_management/cluster-api-about.adoc

```diff
@@ -9,7 +9,7 @@ toc::[]
 :FeatureName: Managing machines with the Cluster API
 include::snippets/technology-preview.adoc[]
 
```

Comment:
This isn't required, but it might be good to unset `:FeatureName:` here, just to be safe.

Reply:
The TP snippet is a bit odd - it actually has the unset in the snippet itself so that we don't need to clean up after it :slightly_smiling_face: